src/presets/_nitro/runtime/nitro-prerenderer.ts

```diff
@@ -1,9 +1,27 @@
 import "#nitro-internal-pollyfills";
+import consola from "consola";
+import { getRequestHeader, getRequestURL, H3Error, isEvent } from "h3";
 import { useNitroApp } from "nitro/runtime";
 import { trapUnhandledNodeErrors } from "nitro/runtime/internal";
 
 const nitroApp = useNitroApp();
 
+nitroApp.hooks.hook("error", (error, context) => {
+  if (
+    isEvent(context.event) &&
+    !(error as H3Error).unhandled &&
```

Comment:
We should also ignore 404/40x (or maybe all non 500x?) errors. When crawling enabled, we hit routes that don't exist or need authentication or are (30x) redirect. We need better cases for this whether verbosity is needed or not.